### DISCUSSION TITLE
Does Czkawka follow symlinks?

### DISCUSSION THREAD

**Discussion by mosiser** [qarmin/czkawka#1384] [Q&A]
Example:

> _**Home**_ folder, containing a **_Pictures_** folder, and a **_Images_** symlink to it.
> Backup (copy) the **_Home_** folder somewhere else.
> 
> Now, if the symlink has a relative path, the **_backed-up-Images_** points correctly to the **_backed-up-Pictures_**.
> Instead, if it has an absolute path, the **_backed-up-Images_** still points to the **_real-Pictures_**
> 
> So, if I search (and remove) duplicates from the **_backed-up-Home_**, is there the risk that I lose original pictures from my **_real-Home_**?

I did a quick test, and it seems Czkawka doesn't follow symlinks (as I hope), while instead Dupeguru, for example, follow the links and delete my original files.
Is it correct? I couldn't find information about this. It should be made explicit.

(If the behavior will change, anytime, please make sure the user is well informed about, because it's an easy way to lose important data...)

Ps: thanks for this great and unique software!

**Comment by nenatap** [comment]
I would like czkawka to follow symlinks. Or to have option to follow (dereference) symlinks and then, if chosen delete the symlink, not the dereferred (original) file. 
I do lots of symlinking around with my collections and this would be useful. For time being, I work around that bt creating a temp folder, where I copy the dereferened files, run czkawka comparision and remove the files. then remove the same named symlinks from the original dir
